mod/wikirate/spec/set/type_plus_right/metric/all_values_spec.rb

```diff
@@ -125,7 +125,7 @@
                                 class: 'header metric-list-header slotter',
                                 href: "/#{url_key}?item=content"\
                                       "&offset=0&limit=20"\
-                                      "&sort_order=desc"\
+                                      "&sort_order=asc"\
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.